### PR TITLE
registe domain

### PR DESCRIPTION
**Pull request by chiragksharma**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

**Review by triach-rold — CHANGES_REQUESTED** [review]
2 JSON files are empty.

File: domains/wtf-chirag-dev
Comment (21Z) [inline domains/wtf-chirag-dev:1]: No records, file is not JSON, invalid.

File: domains/wtf-chirag
Comment (21Z) [inline domains/wtf-chirag:1]: Empty and invalid file

File: domains/mr-random
Comment (21Z) [inline domains/mr-random:1]: Empty and invalid file

**Review by 21Z — CHANGES_REQUESTED** [review]

**Review by iostpa — CHANGES_REQUESTED** [review]
what 21Z and triach-rold said

**Pull request closed without merging**